scripts/docker-summary.sh

```diff
@@ -1,15 +1,19 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-echo "***************Docker Containers Summary***************"
-docker ps -a
-
 DOCKER_IDS=$(docker ps -aq)
 
 for id in ${DOCKER_IDS}
 do
+  echo "***********************************************************"
   echo "***************Docker Container ${id}***************"
+  echo "***********************************************************"
   docker ps -af id=${id} --no-trunc
+  echo "----"
   docker logs ${id} | tail -n 10 || echo "It is not possible to grab the logs of ${id}"
 done
+
```

Comment:
I have moved the `docker ps -a` output to the end of the log when the cocker-compose command fails, so the first info you see at the bottom it is which containers are broken.